types/react-leaflet/react-leaflet-tests.tsx

```diff
@@ -887,3 +888,164 @@ export class GeoJSONExample extends Component<undefined, undefined> {
         </Map>);
     }
 }
+
+const mapEvents: MapEvents = {
+    onclick: (event: L.LeafletMouseEvent) => {
+        console.log(event);
+        console.log(event.target);
+        console.log(event.originalEvent);
+    },
```

Comment:
This is just an idea, but something like the following could be a more compact way of accomplishing the same thing:
```typescript
onclick: ({ target, originalEvent }: L.LeafletMouseEvent) => { }
```